automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -28,9 +28,28 @@ main() {
     echo "Deploy KubeVirt latest stable release"
     ./hack/deploy-kubevirt.sh
 
+    echo "Get skip tests regex"
+    __get_skipped_tests SKIPPED_TESTS
+
     # Run KubeMacPool functional tests
     cd ${TMP_COMPONENT_PATH}
-    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml" make functest
+
+    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml --ginkgo.skip $SKIPPED_TESTS" make functest
+}
+
+function __get_skipped_tests() {
```

Comment:
If you dont mind would you move this code to another file? or at least move this to line 11 to align with other functions

Reply:
I wouldn't want to move it to another file, as it's a very short function that will only be used here.. 
I'll move it to line 11, thanks!

Reply:
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/c9034d2d0005f3c0a2955a6f92251232a9b5d8f0..985f5fc3e5fed1598b977d971dbad37ee43e0550)